test/functional/apps/dashboard/group3/dashboard_state.ts

```diff
@@ -209,210 +192,108 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await PageObjects.dashboard.waitForRenderComplete();
     };
 
-    describe('Directly modifying url updates dashboard state', () => {
-      before(async () => {
-        await PageObjects.dashboard.gotoDashboardLandingPage();
-        await PageObjects.dashboard.clickNewDashboard();
-        await PageObjects.timePicker.setHistoricalDataRange();
-      });
+    describe('for embeddable config color parameters on a visualization', () => {
+      let originalPieSliceStyle = '';
 
-      const changeQuery = async (useHardRefresh: boolean, newQuery: string) => {
-        await queryBar.clickQuerySubmitButton();
-        const currentUrl = await getUrlFromShare();
-        const newUrl = updateAppStateQueryParam(
-          currentUrl,
-          (appState: Partial<SharedDashboardState>) => {
-            return {
-              query: {
-                language: 'kuery',
-                query: newQuery,
-              },
-            };
-          }
-        );
-
-        await browser.get(newUrl.toString(), !useHardRefresh);
-        await PageObjects.dashboard.waitForRenderComplete();
-        const queryBarContentsAfterRefresh = await queryBar.getQueryString();
-        expect(queryBarContentsAfterRefresh).to.equal(newQuery);
-      };
-
-      it('for query parameter with soft refresh', async function () {
-        await changeQuery(false, 'hi:goodbye');
-        await PageObjects.dashboard.expectAppStateRemovedFromURL();
-      });
-
-      it('for query parameter with hard refresh', async function () {
-        await changeQuery(true, 'hi:hello');
+      before(async () => {
         await queryBar.clearQuery();
-        await queryBar.clickQuerySubmitButton();
-        await PageObjects.dashboard.expectAppStateRemovedFromURL();
+        await dashboardAddPanel.addVisualization(PIE_CHART_VIS_NAME);
+        await enableNewChartLibraryDebug();
+        originalPieSliceStyle = (await pieChart.getPieSliceStyle(`80,000`)) ?? '';
       });
 
-      it('for panel size parameters', async function () {
-        await dashboardAddPanel.addVisualization(PIE_CHART_VIS_NAME);
+      it('updates a pie slice color on a hard refresh', async function () {
+        await PageObjects.visChart.openLegendOptionColorsForPie(
+          '80,000',
+          `[data-title="${PIE_CHART_VIS_NAME}"]`
+        );
+        await PageObjects.visChart.selectNewLegendColorChoice('#F9D9F9');
         const currentUrl = await getUrlFromShare();
-        const currentPanelDimensions = await PageObjects.dashboard.getPanelDimensions();
+        await PageObjects.share.closeShareModal();
```

Comment:
Do we need this `closeShareModal` call? It looks like we are already closing it in `getUrlFromShare`